### PR TITLE
Merge chip support diffrent sizes

### LINKED ISSUE #99: Chips should support different sizes
**Component**: Chips (https://scality.github.io/core-ui/?path=/story/chips--default)

**Why this is needed**:
- Chips current has only one size, it should be great to have it in different sizes like Button, Dropdown...

**What should be done**:

**Implementation proposal** (strongly recommended):

**Test plan**:

### PR DESCRIPTION
**Pull request by carlito-scality**
**Component**:Merge

**Description**:
- Merge open source PR https://github.com/scality/core-ui/pull/104

**Design**:
image

**Breaking Changes**:

- [] Breaking Changes

---

Closes: https://github.com/scality/core-ui/issues/99

### CODE REVIEW

**Comment by carlito-scality** [comment]
@edumoreira1506 Hi :) I just pushed a commit to improve some styles. Let me know if it is ok for you.

**Comment by edumoreira1506** [comment]
Nice! It's ok!

**Comment by carlito-scality** [comment]
> Nice! It's ok!

Thanks :)

File: src/lib/components/chips/Chips.component.js [hunk 1]
```diff
@@ -1,144 +1,165 @@
...
 
 const ChipsContainer = styled.div`
   display: inline-flex;
   background-color: ${defaultTheme.grayLighter};
-  border-radius: 15px;
+  .sc-chips-remove {
+    padding-right: 10px;
+    color: ${defaultTheme.white};
+    &:hover {
+      color: ${defaultTheme.gray};
```
Comment (ChengYanJin) [inline src/lib/components/chips/Chips.component.js]: We should not use ${defaultTheme.gray}. If it's a gray Chip, the `remove icon` disappears when you hover.
I will let you address it after merging the dark/light mode PR.

File: src/lib/components/chips/Chips.component.js [hunk 2]
```diff
@@ -1,144 +1,165 @@
...
-          <ChipsButton onClick={onRemove} color={color} variant={variant}>
-            {buttonIcon}
-          </ChipsButton>
-        )}
       </ChipsText>
+      {onRemove && (
+        <Button
+          className="sc-chips-remove"
+          size={size}
+          inverted={true}
```
Comment (ChengYanJin) [inline src/lib/components/chips/Chips.component.js]: I guess we don't need `inverted` prop in Chips component. :)

**Review by ChengYanJin — CHANGES_REQUESTED** [review]

**Review by ChengYanJin — APPROVED** [review]
LGTM!

**Comment by carlito-scality** [comment]
@bert-e approve

**Pull request merged**